src/integrationTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerControllerTest.java

```diff
@@ -23,13 +23,7 @@
 import static org.mockito.BDDMockito.given;
 import static org.mockito.Mockito.inOrder;
 
-@SpringBootTest(classes = {
```

Comment:
This is a way to save up some time when loading the context when running the IT. However, it's specific for this test. Other tests might need different context.